src/SFA.DAS.ApprenticeCommitments/Data/Models/ApprenticeCommitmentsDbContext.cs

```diff
@@ -20,23 +21,37 @@ public ApprenticeCommitmentsDbContext(DbContextOptions<ApprenticeCommitmentsDbCo
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<Apprentice>(a =>
-                {
-                    a.Property(e => e.Id).ValueGeneratedOnAdd();
-                    a.Property(e => e.CreatedOn).ValueGeneratedOnAdd();
-                    a.Property(e => e.CreatedOn).Metadata.SetAfterSaveBehavior(PropertySaveBehavior.Ignore);
-                });
+            {
+                a.ToTable("Apprentice");
```

Comment:
This is a just question out of curiosity, but does it need to ToTable("Apprentice") as the name matches the entity. I must drill down into the code one day to see what it does

Reply:
EF uses plural names by default.  I removed the `[Table]` attribute on the `Apprentice` class that specified a singular name, and moved it into the modelling code here.